### ISSUE TITLE
Need a way to verify that code added an event handler to an event

### ISSUE THREAD

**Issue by hughbe** [moq/moq4#825]
E.g. I have the product code

```
IDesignerEventService service = ...;
service.ActiveDesignerChanged += handler;
```

I need to verify that we called `+=`

**Comment by stakx** [comment]
... and you're not the first. Someone will have to implement this feature or it's not going to happen...

**Comment by stakx** [comment]
@lepijohnny is currently working on this in #857.

**Comment by stakx** [comment]
Resolved in #857. You'll be able to use `Setup[Add|Remove]` and `Verify[Add|Remove|All]` for event subscription and unsubscription from version 4.13.0 onwards.

Thanks @lepijohnny for having taken care of this long-standing feature request. (There have been a few issues in the past asking for this same feature.)
